### ISSUE TITLE
[$250] Receipts emailed to receipts@expensify.com are not added to Search

### ISSUE THREAD

**Issue by m-natarajan** [Expensify/App#53272]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 9.0.68-0
**Reproducible in staging?:** y
**Reproducible in production?:** y
**If this was caught on HybridApp, is this reproducible on New Expensify Standalone?:**
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** @mattmoore
**Slack conversation** (hyperlinked to channel name): [expensify-retain](https://expensify.slack.com/archives/C07NZ8B1VTQ/p1732730352701389?thread_ts=1732729936.732489&cid=C07NZ8B1VTQ)

## Action Performed:

1. Create new account (no workspace)
2. Email receipts to [receipts@expensify.com](mailto:receipts@expensify.com)
3. Wait for expenses to appear in Self-DM
4. Navigate to Search

## Expected Result:
Expenses are added to Search
## Actual Result:
Expenses are not added to Search
## Workaround:
unknown
## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Standalone
- [ ] Android: HybridApp
- [ ] Android: mWeb Chrome
- [ ] iOS: Standalone
- [ ] iOS: HybridApp
- [ ] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

<details>
  <summary>Add any screenshot/video evidence</summary>

  
</details>

https://github.com/user-attachments/assets/36a922c6-6767-4016-b830-5db8bd3f25d0

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~021875196386780124159</li>
        <li>Upwork Job ID: 1875196386780124159</li>
        <li>Last Price Increase: 2025-01-24</li>
    </ul>
</details>

**Comment by sonialiap** [comment]
another issue with self-DM expenses, these are added through app rather than email https://github.com/Expensify/App/issues/53290

**Comment by sonialiap** [comment]
oh hold

**Comment by sonialiap** [comment]
I'm OOO Dec 16-20, but I don't think a buddy is needed here

**Comment by sonialiap** [comment]
on hold for https://github.com/Expensify/App/issues/52193

**Comment by sonialiap** [comment]
https://github.com/Expensify/App/issues/52193 deployed to prod, ready for a retest

**Comment by kavimuru** [comment]
Still reproduced 

https://github.com/user-attachments/assets/9f58a61c-4ca4-4cc7-a561-d8166f1b1257

**Comment by c3024** [comment]
Waiting for proposals!

**Comment by narefyev91** [comment]
Hi, I'm Nicolay from Callstack - expert contributor group - and I would like to work on this issue.

**Comment by narefyev91** [comment]
Based on my findings so far:
1. I was able to submit receipt via email and get it visible

<img width="1101" alt="Image" src="https://github.com/user-attachments/assets/8f80dead-21cb-425d-9dad-0e3b96c9abd5" />

2. In the Onyx data i get latest transaction data (as well in OpenReport API command):

>     {
      "onyxMethod": "mergecollection",
      "key": "transactions_",
      "value": {
        "transactions_8271299572509833172": {
          "amount": 0,
          "bank": "",
          "billable": false,
          "cardID": 0,
          "cardName": "Cash Expense",
          "cardNumber": "",
          "category": "",
          "comment": "",
          "created": "2025-01-17",
          "currency": "EUR",
          "filename": "w_cd23452bedb5646c5baf32e44fe16fe409298298.pdf",
          "inserted": "2025-01-17 10:49:51",
          "managedCard": false,
          "merchant": "(none)",
          "modifiedAmount": -2405,
          "modifiedCreated": "2023-12-12",
          "modifiedCurrency": "EUR",
          "modifiedMerchant": "Supermarket",
          "originalAmount": 0,
          "originalCurrency": "",
          "parentTransactionID": "",
          "posted": "",
          "receipt": {
            "receiptID": 3849379947447051,
            "state": "SCANCOMPLETE",
            "source": "https:\/\/www.expensify.com\/receipts\/w_cd23452bedb5646c5baf32e44fe16fe409298298.pdf"
          },
          "reimbursable": true,
          "reportID": "",
          "status": "Posted",
          "tag": "",
          "transactionID": "8271299572509833172",
          "hasEReceipt": false
        }
      }
    },

3. When open Search - we fire API call - and probably inside we should get data - but it returns empty

> {
  "accountID": 18996498,
  "authToken": "my_token",
  "email": "m51660517@gmail.com",
  "onyxData": [
    {
      "key": "snapshot_243428839",
      "onyxMethod": "set",
      "value": {
        "data": [
          
        ],
        "search": {
          "columnsToShow": {
            "shouldShowCategoryColumn": false,
            "shouldShowTagColumn": false,
            "shouldShowTaxColumn": false
          },
          "hasMoreResults": false,
          "hasResults": true,
          "offset": 0,
          "status": "all",
          "statusToShow": {
            "expense": {
              "all": true,
              "approved": false,
              "drafts": false,
              "outstanding": true,
              "paid": true
            }
          },
          "type": "expense"
        }
      }
    }
  ],
  "httpCode": 200,
  "jsonCode": 200,
  "authResponseMessage": "200 OK",
  "requestID": "9036033f499ddd54-VNO"
}

4. If i manually add self dm expense - if will get it in search response - but still my expense from email will not be come from API side (on screenshot - i have 2 transactions - but in search response just one manual)

<img width="1717" alt="Image" src="https://github.com/user-attachments/assets/d49e49a5-feb7-4b2d-b8a8-2423abaf82c0" />

@c3024 let me know wdyt - probably we need some from BE team to help us identify - why API response does not have needed info

**Comment by c3024** [comment]
@narefyev91 , yes, `Search` response does not include these receipts sent to receipts@expensify.com. 

@muttmuure , this needs a backend fix.

**Comment by sonialiap** [comment]
Thanks! Adding the internal label

**Comment by sonialiap** [comment]
asking for a retest

**Comment by mvtglobally** [comment]
Issue not reproducible during KI retests. (First week)

**Comment by sonialiap** [comment]
I emailed in a receipt last week and it showed up in Search. I also cannot reproduce
